scalableminds/webknossos
# Update password hashing library

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Create users with passwords before applying this merge, ensure that you can still login afterward.
- ~~Check if front-end still works with CSP @philippotto perhaps you can take a look at the CSP? I had to enable unsafe-inline and unsafe-eval to get WK to run, which significantly weakens security. Don't know if the code can be adjusted to make it work without them. If inline scripts need to be supported but don't change often/dynamically, nonces or hashes can be used to enable only specific inline scripts.~~ -> Content Security Policy moved to #7345 

------
(Please delete unneeded items, merge only when none are left open)
